app/javascript/src/patternflyStyles/pf4Base.css

```diff
@@ -0,0 +1,6 @@
+@import '~@patternfly/patternfly/patternfly-variables.css';
```

Comment:
This file contains the basic PF4 styles, it can be imported in any pf4 component as starting point